delivery/src/main/java/com/nangman/delivery/presentation/DeliveryController.java

```diff
@@ -0,0 +1,57 @@
+package com.nangman.delivery.presentation;
+
+import com.nangman.delivery.application.dto.request.DeliveryPostRequest;
+import com.nangman.delivery.application.dto.request.DeliveryPutRequest;
+import com.nangman.delivery.application.dto.response.DeliveryResponse;
+import com.nangman.delivery.application.service.DeliveryService;
+import java.util.UUID;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestHeader;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/deliveries")
+@RequiredArgsConstructor
+public class DeliveryController {
+
+    private final DeliveryService deliveryService;
+
+    @PostMapping
+    public ResponseEntity<DeliveryResponse> createDelivery(@RequestBody DeliveryPostRequest request) {
```

Comment:
```suggestion
    public ResponseEntity<DeliveryResponse> createDelivery(@Valid @RequestBody DeliveryPostRequest request) {
```
validation이 필요할것 같아요!